gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat May 20 14:19:53 MST 2017
+#Thu Jun 01 10:45:08 MST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-rc-1-bin.zip
```

Comment:
It's great that there's a GA release of Gradle 4.  FWIW, I don't feel that updating the version of the Gradle Wrapper -- even a major version -- would in any way violate (on it's own... all else being equal...) the principals of Semantic Versioning.

We would (I believe) be able to make an update like that w/o breaking any APIs, configs, database schemas, etc.